### PR TITLE
Filled toast

### LINKED ISSUE #3641: msgs for filled order toast
If filling entire order would say 20 of 20					
if filling partial, the next trade would be of 8....for example next trade could show filling 8 of 8 (remaining balance of initial 20)					
IF partial filling order and balance gets added to order book 2 toast messages needed 1) Filled toast; 2) open order for balance toast					

If open order getting filled
- to find original quantity:
    - it is full amount, if in userOpenOrders the matching orderId isn’t found.
    - If it is, then the original quantity is the leftover amount + the amount filled. 

If filling open order
- don’t need to find original amount
- Always says they filled the full amount 
- Need to find open order alert, if it was completely filled get rid of it, if it wasn’t change balance to how much was done without fill

see excel sheet
https://docs.google.com/spreadsheets/d/1Q18_TKEg9lcSrZX8eWcKt26IjTAc-KNMfBn8oaKBrYc/edit?ts=5d7969a8#gid=0v

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/3641

### CODE REVIEW

File: packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts [hunk 1]
```diff
@@ -152,15 +153,38 @@ export default function setAlertText(alert: any, callback: Function) {
...
+              // creator
+              const orders = userOpenOrders[alert.params.market];
+              const outcome = new BigNumber(alert.params.outcome).toString()
+              const foundOrder = orders[outcome] && orders[outcome][alert.params.orderType] && orders[outcome][alert.params.orderType][alert.params.orderId];
+              if (foundOrder) {
+                originalQuantity = createBigNumber(foundOrder.originalFullPrecisionAmount);
+              }
+            } else {
+              // filler
+              updatedOrderType = alert.params.orderType === 0 ? 1 : 0;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts]: use constants here for BUY_INDEX | SELL_INDEX not great names but use a constant

File: packages/augur-ui/src/modules/alerts/components/alerts-view.styles.less [hunk 2]
```diff
@@ -91,26 +91,24 @@
 .dismissContainer {
   align-items: flex-end;
 
-  background: @color-contrast-background-theme;
+  background: #201E22;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/alerts/components/alerts-view.styles.less]: stuff for colors effort to clean up, right?
Need to talk to John about this color. @JohnDanz

File: packages/augur-ui/src/modules/alerts/selectors/alerts.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const selectInfoAlertsAndSeenCount = createSelector(
         .filter(
           alert =>
             (alert.networkId === networkId.toString() &&
-              alert.universe === universe.id) ||
+              alert.universe === universe.id && (alert.status === "Failure" || alert.status === "Success")) ||
```
Comment (bthaile) [inline packages/augur-ui/src/modules/alerts/selectors/alerts.ts]: need constants for Failure and Success, 
can TXEventName.Failure and TXEventName.Success be used here?

File: packages/augur-ui/src/modules/transactions/actions/construct-relay-transaction.ts [hunk 4]
```diff
@@ -22,20 +22,20 @@ export const constructRelayTransaction = (tx: any) => (
...
-        params: unpackedParams,
-        status,
-        // @ts-ignore
-        title: unpackedParams.type,
-        description: "",
-        linkPath: makePath(TRANSACTIONS, false),
-        to: tx.data.to,
-      }),
-    );
+    // dispatch(
```
Comment (bthaile) [inline packages/augur-ui/src/modules/transactions/actions/construct-relay-transaction.ts]: just remove this commented code.

File: packages/augur-ui/src/modules/alerts/selectors/alerts.ts (hunk 3 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/alerts/selectors/alerts.ts]: was causing a crash when i used those but /constants has some similar ones

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 😄

 This link will always point to this PR's latest changes: https://peek.run/0elhppz

**Review by bthaile — APPROVED** [review]
let's get this in so we can test how it integrates.

**Comment by bthaile** [comment]
I accidentally merged in a PR I was reviewing. it will be fine

**Comment by bthaile** [comment]
Just noting here. seeing many alerts when sweeping the book, touching base with Chwy

image

Created ticket to address this flooding effect
<https://github.com/AugurProject/augur/issues/3730>

**Pull request merged**